ws/templates/for_templatetags/feedback_table.html

```diff
@@ -19,7 +19,7 @@ <h3 id='feedback'><i class="fas fa-comments"></i> Feedback </h3>
             <span class="label label-danger">Flaked!</span>
           {% endif %}
 
-          {{ feedback.comments }}
+          {{ feedback.comments|linebreaks }}
```

Comment:
This converts the comments to paragraphs, which is different than the JavaScript + CSS solution you have in the Angular template. For example:

### Django
<img width="784" alt="image" src="https://user-images.githubusercontent.com/901169/51158518-bc211d00-1839-11e9-8a89-a91f9726c428.png">

### Angular
<img width="335" alt="image" src="https://user-images.githubusercontent.com/901169/51158560-ea9ef800-1839-11e9-95c7-5224b0b7dda3.png">